test/unit_tests/braket/tasks/test_gate_model_quantum_task_result.py

```diff
@@ -373,3 +373,35 @@ def test_calculate_ir_results_value_error():
     measured_qubits = [0]
     measurements = np.array([[0]])
     GateModelQuantumTaskResult._calculate_result_types(ir_string, measurements, measured_qubits)
+
+
+def test_hash_result_types():
+    expectation_1 = jaqcd.Expectation(targets=[1, 2], observable=["x"])
+    expectation_2 = jaqcd.Expectation(observable=["x"], targets=[1, 2])
+    expectation_3 = jaqcd.Expectation(observable=["x"], targets=[2, 1])
+    expectation_4 = jaqcd.Expectation(
+        observable=[[[[0, 0], [1, 0]], [[1, 0], [0, 0]]], [[[1, 0], [1, 0]], [[1, 0], [-1, 0]]]],
+        targets=[1, 2],
+    )
+    expectation_5 = jaqcd.Expectation(
+        observable=[[[[0, 0], [1, 0]], [[1, 0], [0, 0]]], [[[1, 0], [1, 0]], [[1, 0], [-1, 0]]]],
+        targets=[1, 2],
+    )
```

Comment:
Probably being paranoid, but also add a test case for Hermitians with a a bunch of digits after the decimal, just for peace of mind.

Reply:
done